patches/third_party-rust-hashbrown-v0_15-BUILD.gn.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/third_party/rust/hashbrown/v0_15/BUILD.gn b/third_party/rust/hashbrown/v0_15/BUILD.gn
+index 8595d7b24c4bb087c6dc04d811533e29fe50e593..b00d59e3e19325780e8d374d19432096c7521f64 100644
+--- a/third_party/rust/hashbrown/v0_15/BUILD.gn
++++ b/third_party/rust/hashbrown/v0_15/BUILD.gn
+@@ -43,7 +43,19 @@ cargo_crate("lib") {
+     "//third_party/rust/chromium_crates_io/vendor/hashbrown-v0_15/src/util.rs",
+   ]
+   inputs = []
+-
++  deps = [
```

Comment:
this should follow normal patching standards

Reply:
Updated in https://github.com/brave/brave-core/pull/28471/commits/4bdba2966045771e5182645d688408a814fce4b2